packages/scss/src/components/table/mods.scss

```diff
@@ -53,6 +53,40 @@
 @mixin card {
 	border-top: 0;
 
+	:where(.table-head, .table-body, .table-foot) {
+		&:first-child {
```

Comment:
i don't understand why target first table-head ?

Reply:
To manage tables with or without table-heads.